pywhyllm/suggesters/validation_suggester.py

```diff
@@ -93,29 +76,29 @@ def request_negative_controls(
 
                 with system():
                     lm += f"""You are an expert in the {domain_expertise} and are 
-        studying the {analysis_context}. You are using your domain knowledge to help understand the negative 
-        controls for a causal model that contains all the assumptions about the {analysis_context}. Where a causal 
-        model is a conceptual model that describes the causal mechanisms of a system. You will do this by answering 
-        questions about cause and effect using your domain knowledge in the {domain_expertise}."""
+            studying the {analysis_context}. You are using your domain knowledge to help understand the negative 
```

Comment:
are these extra spaces intended? Would they add unnecessary spaces in the final prompt?